### PR TITLE
RDoc-2190 [Node.js] Client API > Session > Updating Entities

### PR DESCRIPTION
**Pull request by Danielle9897**
Related issue:
https://issues.hibernatingrhinos.com/issue/RDoc-2190/Nodejs-Client-API-Session-Updating-Entities

### CODE REVIEW

**Comment by Danielle9897** [comment]
@ml054 
For Node.js - the relevant files to be reviewed are:
```
Documentation/5.2/Raven.Documentation.Pages/client-api/session/updating-entities.js.markdown
Documentation/5.2/Samples/nodejs/ClientApi/Session/updateDocuments.js
```

File: Documentation/5.2/Samples/nodejs/ClientApi/Session/updateDocuments.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import {DocumentStore} from "ravendb";
+
+const documentStore = new DocumentStore();
+
+async function updateDocuments() {
+    {
+        //region sample_document
+        // Sample company document structure
+        class address {
```
Comment (ml054) [inline Documentation/5.2/Samples/nodejs/ClientApi/Session/updateDocuments.js:9]: why lower case?

File: Documentation/5.2/Samples/nodejs/ClientApi/Session/updateDocuments.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        //region query-company-and-update
+        const session = documentStore.openSession();
+
+        // Query: find companies with the specified postalCode
+        const matchingCompanies = await session.query({collection: "companies"})
+            .whereEquals("address.postalCode", "12345")
+            .all();
+
+        // Update the postalCode for the resulting company documents
+        matchingCompanies.forEach(c => c.address.postalCode = "TheNewPostalCode")
```
Comment (ml054) [inline Documentation/5.2/Samples/nodejs/ClientApi/Session/updateDocuments.js:47]: missing ;

Comment (Danielle9897) [reply Documentation/5.2/Samples/nodejs/ClientApi/Session/updateDocuments.js:47]: done

File: Documentation/5.2/Samples/nodejs/ClientApi/Session/updateDocuments.js (hunk 1 above)
Comment (Danielle9897) [reply Documentation/5.2/Samples/nodejs/ClientApi/Session/updateDocuments.js:9]: done

**Review by ml054 — APPROVED** [review]

File: Documentation/5.2/Raven.Documentation.Pages/client-api/session/.docs.json [hunk 3]
```diff
@@ -56,14 +56,9 @@
     },
     {
         "Path": "updating-entities.markdown",
-        "Name": "Updating Entities",
+        "Name": "Update Documents",
```
Comment (ppekrol) [inline Documentation/5.2/Raven.Documentation.Pages/client-api/session/.docs.json:59]: In session those are Entities

File: Documentation/5.2/Raven.Documentation.Pages/client-api/session/updating-entities.dotnet.markdown [hunk 4]
```diff
@@ -0,0 +1,65 @@
+# Session: Update Documents
```
Comment (ppekrol) [inline Documentation/5.2/Raven.Documentation.Pages/client-api/session/updating-entities.dotnet.markdown:1]: Entities

**Pull request merged**